core/web/resolver/vrf.go

```diff
@@ -133,11 +133,19 @@ func (r *DeleteVRFKeySuccessResolver) Key() VRFKeyResolver {
 
 type DeleteVRFKeyPayloadResolver struct {
 	key vrfkey.KeyV2
-	err error
+	NotFoundErrorUnionType
 }
 
 func NewDeleteVRFKeyPayloadResolver(key vrfkey.KeyV2, err error) *DeleteVRFKeyPayloadResolver {
-	return &DeleteVRFKeyPayloadResolver{key, err}
+	var e NotFoundErrorUnionType
+
+	if err != nil {
```

Comment:
And here.

Reply:
same reason as stated before on why this needs to happen